packages/api-gen/src/patches.ts

```diff
@@ -7,7 +7,12 @@ export const patches = {
      * Fixes the problem  with schema >=5.0 where ContextTokenResponse is not defined
      */
     name: "ContextTokenResponse",
-    patch: (schema: Object) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    patch: (schema: any) => {
```

Comment:
couldn't find proper type for schema, will get back to this line in another PR, but didn't want to hole whole typecheck because of that

Reply:
fixed